plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1132,6 +1134,11 @@ public boolean configure(final String name, final Map<String, Object> params) th
             s_logger.trace("Ignoring libvirt error.", e);
         }
 
+        // Enable/disable IO driver for Qemu (in case it is not set CloudStack can also detect if its supported by qemu)
+        // Do not remove - switching it to AgentProperties.Property may require accepting null values for the properties default value
+        String enableIoUringConfig = (String) params.get(ENABLE_IO_URING_PROPERTY);
+        enableIoUring = isIoUringEnabled(enableIoUringConfig);
+
```

Comment:
one small suggestion if you want to log if the IO uring is enabled/disabled

Reply:
Yes please, let's do the proper logging.

Reply:
Done, thanks